### DISCUSSION TITLE
timezone warning

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#3147] [Ideas] [answered]
maybe there should be a warning shown inside the mastercontainer, if either TZ is set and/or /etc/localtime is mounted

**Comment by szaimen** [comment]
Does https://github.com/nextcloud/all-in-one/blob/ee86f23856d0119dfcdd7aad179195668f270da4/Containers/mastercontainer/start.sh#L308 not overwrite this? If not I agree that we could add a check if the timezone is set and if /etc/localtime is mounted

**Reply by Zoey2936** [reply]
I thought /etc/localtime has a higher priority, but not sure

**Reply by szaimen** [reply]
I tested it and exporting TZ seems to overwrite the setting... Should we still add a warning for that or WDYT?

**Reply by Zoey2936** [reply]
I woukld still say yes

**Comment by szaimen** [comment, marked as answer]
Will be done in https://github.com/nextcloud/all-in-one/pull/3153
